padre_meddea/calibration/calibration.py

```diff
@@ -320,3 +320,162 @@ def read_calibration_file(calib_filename: Path):
     # if can't read the file
 
     return None
+
+
+def concatenate_daily_fits(
+    files_to_combine: list[Path], existing_file: Path = None, outfile: Path = None
+) -> Path:
+    """
+    Concatenate multiple FITS files into a single daily FITS file, properly combining headers and data.
+
+    Parameters
+    ----------
+    files_to_combine : list of Path
+        List of FITS files to combine. Assumed to have the same structure.
+    existing_file : Path, optional
+        Existing daily FITS file to append to.
+    outfile : Path, optional
+        Output file path. If None, will generate automatically.
+
+    Returns
+    -------
+    output_file : Path
+        Path to the concatenated daily FITS file.
+    """
+    from astropy.time import Time
+    import numpy as np
+    import logging
+
+    log = logging.getLogger(__name__)
+
+    # Validate input
+    if not files_to_combine and not existing_file:
```

Comment:
the `files_to_combine` does not have a default value so I think you would get an error without this code.